en/rest/hooks.mdown

```diff
@@ -0,0 +1,671 @@
+# Hooks
+
+You can create, update, list or delete all your cloud code webhooks via the Hooks API,
+in addition to being able to do so through the parse website.
+
+Hooks API requires that the user provides `Application-Id` and `Master-Key` in the request headers.
+
+There are two kinds of cloud code webhooks: cloud function hooks and object trigger hooks.
+
+Cloud functions are functions that run in the cloud and allow you to build functions
+common to all platforms. They can be changed without redeploying your app.
+For more details please read [cloud code functions](https://parse.com/docs/js/guide#cloud-code-cloud-functions).
+
+Object triggers are invoked whenever you save or delete a parse object.
+Triggers are commonly used to validate your objects before saving them.
+For more details please read [cloud code triggers](https://parse.com/docs/js/guide#cloud-code-beforesave-triggers).
+
+## Fetch functions
+To fetch the list of all cloud functions and cloud function webhooks you deployed or created, use
+
+```bash
+curl -X GET \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "Content-Type: application/json" \
+  https://api.parse.com/1/hooks/functions
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('GET', '/1/hooks/functions', '', {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+Sample output
+```json
+{
+  "results": [
+    { "functionName": "sendMessage", "url": "https://api.example.com/sendMessage" },
+    { "functionName": "sendMessage" },
```

Comment:
Could you put this explanation in the documentation?  This was confusing to me as well.  Also link to the documentation elsewhere that actually explains the difference between webhook and cloud function.